pkg/sdk/data_metric_function_references_gen_test.go

```diff
@@ -0,0 +1,51 @@
+package sdk
+
+import "testing"
+
+func TestDataMetricFunctionReferences_GetForEntity(t *testing.T) {
```

Comment:
there are 5 different validation results + 1 correct validation and here are only 5 tests, so by "my math" at least one is missing

Reply:
Added.